src/main/java/calculator/domain/Operation.java

```diff
@@ -0,0 +1,55 @@
+package calculator.domain;
+
+import static calculator.domain.ErrorMessage.*;
+
+public class Operation {
+    private static final String ADD_SYMBOL = "+";
+    private static final String SUBSTRACT_SYMBOL = "-";
+    private static final String MULTIPLY_SYMBOL = "*";
+    private static final String DIVIDE_SYMBOL = "/";
+    private int num1 = 0;
+    private int num2 = 0;
+    private String symbol = "";
```

Comment:
```suggestion
    private final int num1;
    private final int num2;
    private final String symbol;
```
1. 불변하게 설계하면 안정적인 객체를 만들 수 있습니다 :) 
2. 초기값이 유의미하지 않다면 혼동을 주기 때문에 제거하는 것이 좋습니다.